conf/gb/united_kingdom03.yml

```diff
@@ -356,10 +332,6 @@ years:
     date: '2021-01-04'
     names:
       en: New Year's Day
-  - public_holiday: true
-    date: '2021-04-05'
```

Comment:
🤔 branch name says "update-scotland-easter-monday-2020" but changeset doesn't change 2020...? Looks like it jumps straight from 2016 to 2021?

Reply:
Oh yeah, good point. I checked out the branch before looking at the file. I meant to remove all Easter Mondays and that is what I've done, but now I'm wondering why the customer got in touch to say we were citing (this) Monday as a public holiday in Scotland when that can't be true...

Also looking at various council websites in Scotland, this is extremely variable accross regions in Scotland 🤔

Reply:
Hm, maybe check whether the company actually has `united_kingdom03` set as their hols.

And then ***don't say anything here*** about that because this is a public repo 🔐

Reply:
@tmcdb what's happening with this one?